frontend/src/pages/modelRegistry/ModelRegistryCoreLoader.tsx

```diff
@@ -55,6 +58,22 @@ const ModelRegistryCoreLoader: React.FC<ModelRegistryCoreLoaderProps> =
       preferredModelRegistry?.metadata.name,
     ]);
 
+    if (!modelRegistryNamespace) {
+      return (
+        <ApplicationsPage
+          loaded
+          empty={false}
+          loadError={new Error('No registries namespace could be found')}
+          loadErrorPage={
+            <RedirectErrorState
+              title="Could not load component state"
+              errorMessage="No registries namespace could be found"
+            />
+          }
+        />
```

Comment:
You aren't actually rendering what you pass to `new Error` here, you honestly could just do the same thing I suggested for AreaContext and omit `loadError` and `loadErrorPage` and instead pass the RedirectErrorState as children.